charts/ozone/templates/datanode/datanode-statefulset.yaml

```
@@ -40,6 +40,9 @@ spec:
     metadata:
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/ozone-configmap.yaml") . | sha256sum }}
+        {{- with .Values.datanode.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
```

```suggestion
        {{- with .Values.datanode.podAnnotations }}
          {{- toYaml . | nindent 8 }}
        {{- end }}
```
nit: I suggest adding two spaces to follow the project's formatting approach.